### DISCUSSION TITLE
Vaultwarden - try to upgrade with more Power

### DISCUSSION THREAD

**Discussion by dancgn** [community-scripts/ProxmoxVE#367] [Q&A]
Hello there,

after reading about a couple of security updates I’ve want to upgrade the container. So I give it max Power (4 CPUs, 8GB RAM) and it gave me an error about not enough RAM. I gave more than 7 GB, shut down all other containers and even restart the node. Nothing helps.

Thanks for any tip.
Dan

**Comment by MickLesk** [comment]
Can you share a Screenshot?

I tested it, and it works with every Container ive tested.

  if [[ "$current_ram" -lt "$var_ram" ]] || [[ "$current_cpu" -lt "$var_cpu" ]]; then

Says: current_ram LESS THEN var_cpu
As Example 
8192 LESS THEN 8192 = Nope. Ignored. 

If 8191 LESS THEN 8192 = Error Dialogue

**Reply by dancgn** [reply]
<img width="445" alt="Bildschirmfoto 2024-11-19 um 21 02 53" src="https://github.com/user-attachments/assets/1190894b-7721-4d82-aca5-19610087888d">
<img width="780" alt="Bildschirmfoto 2024-11-19 um 21 02 31" src="https://github.com/user-attachments/assets/99fb9fc5-ee14-4328-868f-60f03b5f7c44">

Here you go

**Reply by MickLesk** [reply]
Nothing? Interessting.

Whats your Output Here:
 echo "RAM: $(free -m | awk '/^Mem:/{print $2}') MB"

Have you this Problem on other LXC's too?

**Reply by dancgn** [reply]
Same, he said nothing. And, no. Only this container has this problem. 
<img width="630" alt="Bildschirmfoto 2024-11-20 um 09 48 15 jpeg" src="https://github.com/user-attachments/assets/b9a30bca-38e4-4eaa-904a-6346b123b177">

**Reply by MickLesk** [reply]
whats defect with your lxc :D

**Reply by dancgn** [reply]
I think I’m special. 

The Best would be to build a new lxc with Vaultwarden I guess.

**Reply by dancgn** [reply]
So, I just make a new one. Still same error. 

But it's the newest Version for now.

**Reply by MickLesk** [reply]
Same LXC ID ?

**Reply by dancgn** [reply]
No, different. Same IP, but doesn’t matter I guess
